.github/vale/styles/Aiven/capitalization_headings.yml

```diff
@@ -97,7 +99,11 @@ exceptions:
   - TimescaleDB
   - Transport Layer Security
   - United States
+  - UpCloud
   - Virtual Private Cloud
+  - VPC
+  - VM
+  - VMs
```

Comment:
Same comment as before, if we're not using `VPC`, `VM`, or `VMs` in titles, these rows shouldn't be needed